python/ray/ml/predictors/integrations/tensorflow/tensorflow_predictor.py

```diff
@@ -136,9 +136,14 @@ def build_model(self):
                 data = data[feature_columns]
             data = data.values
         else:
-            data = data[:, feature_columns]
```

Comment:
if `feature_columns` is None, this will automatically add one more dimension to the numpy array, which is probably not intended.